### DISCUSSION TITLE
Whats new update for 2022

### DISCUSSION THREAD

**Discussion by APKROB** [skyline-emu/skyline#307] [General]
Hi guys im the number fan of skyline where can i donate for some little help. 
Anyway im just looking for a new update for this Emulator. The sonic mania is now playable . Thanks for advance. ☺️

**Comment by proganime1200** [comment]
Up same im thinking of donating

**Reply by sauradipdebnath** [reply]
In the latest update there is no graphic output

**Comment by JerryM33** [comment]
Update the version number🤣
